app/views/static_pages/about.html.erb

```diff
@@ -1,10 +1,12 @@
+<% provide :title, t(".about") %>
 <h1>About</h1>
 <p>
-  The <a href="https://www.railstutorial.org/"><em>Ruby on Rails
-  Tutorial</em></a> is a
+  The <a href="https://www.railstutorial.org/"><em><%= t ".about" %></em></a> is a
```

Comment:
- Thay toàn bộ thẻ a thành link_to
- Dùng gem config cho "https://www.railstutorial.org/" vào Settings nhé em